css/style.css

```diff
@@ -85,6 +85,16 @@ body {
   position: relative;
 }
 
+.border {
+  height: 3px;
+  width: 30%;
+  background-color: var(--secondary-color);
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 10px;
+  margin-bottom: 10px;
+}
```

Comment:
You can optimise margin to use short hand property, see margin short hand here: https://www.w3schools.com/Css/css_margin.asp